llvm/test/CodeGen/Mips/Fast-ISel/underaligned-load-store.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc < %s -march mips -fast-isel -relocation-model pic | FileCheck %s -check-prefixes=MIPS
```

Comment:
Can we use update_llc_test_checks.py to generate the CHECK lines?

Reply:
Done; wasn't aware of this tool.